.cbmc-batch/jobs/aws_array_list_pop_back/Makefile

```diff
@@ -12,14 +12,18 @@
 # limitations under the License.
 
 ###########
-CBMC_UNWINDSET =
+include ../Makefile.aws_array_list
+
+# This bound allow us to reach full coverage rate
```

Comment:
🍓 s/allow/allows/ and we may want to say "100% statement coverage" instead of "full coverage rate"